### DISCUSSION TITLE
Não consigo cortar madeira

### DISCUSSION THREAD

**Discussion by WesleyGarotinho** [arkypita/LaserGRBL#1746] [General]
Gostaria de pedir uma ajuda pois desde que minha máquina chegou, não consigo fazer uso dela de forma correta... nunca ela foi capaz de efetuar um corte e eu fazer meus projetos como planejado. Alguém poderia me ajudar por gentileza. ?

**Comment by Tribo-2dot0** [comment]
Coloque mais informações, como,  qual é a máquina e o módulo do laser, qual a placa controladora... está ajustando o foco corretamente? Qual a  potencia e velocidade que tem usado? O laser se mexe? Ele acende em algum momento?

**Reply by WesleyGarotinho** [reply]
obrigado pelo rápido retorno... sim o laser está normal, se movimenta por toda a armação e está focado sim... o que aparenta estar ocorrendo é, não haver força suficiene para efetuar o corte, mesmo passando por várias vezes sobre o desenho. Ex: se eu tentar cortar um MDF 3mm, passando com a velocidade de 200 por 11 vezes na potência 100%, não está funcionando, acredito ser passadas mais que suficientes, esperava ter um retorno melhor do equipamento pela potência denominada ( 20W - 450nm )

**Reply by WesleyGarotinho** [reply]
![WhatsApp Image 2022-02-09 at 15 01 49 (1)](https://user-images.githubusercontent.com/99302450/153265832-c8e0440c-3770-4a91-bc8e-14596a73752f.jpeg)
![WhatsApp Image 2022-02-09 at 15 01 49](https://user-images.githubusercontent.com/99302450/153265834-bf0e8d4c-17e3-4974-bcba-4a917ea71028.jpeg)
![WhatsApp Image 2022-02-09 at 15 01 50 (1)](https://user-images.githubusercontent.com/99302450/153265835-0c8cce6d-4668-4f86-81ec-4cbd80d960d1.jpeg)

**Comment by Tribo-2dot0** [comment]
pois então, tenho um laser de 30W e tenho dificuldade de cortar MDF de 3mm... uma por que meu módulo é de foco fixo, o foco dele está em 20mm, o que é ótimo para gravação (para corte é melhor 40mm de distância, isso ira alongar a faixa mais estreita do laser), outra coisa é o tipo de MDF, uns que parecem mais compactados corto em um ou dois passes, outros não corto nem com 5 passes, mais que isso ja vai ter tanta carbonização que cria uma barreira contra o laser, para isso deve-se usar assistencia de ar, coisa que também não tenho, o ar comprimido soprado no ponto do corte durante o percurso, ajuda muito...
O seu foco, pelo que vi, é ajustável, tente deixar a uma distância de 40mm que ja vai ajudar, mas se quer cortar mesmo, adquira um kit de assistencia de ar, você encontra os compressores à venda no AliExpress.

https://pt.aliexpress.com/item/1005002882718537.html

https://pt.aliexpress.com/item/1005003671405685.html

**Comment by WesleyGarotinho** [comment]
Entendi... pode ser a distância então, farei esse ajuste pra 40mm. Pra ter
idéia, um camarada me emprestou um compressorzinho destes e mesmo assim não
surtiu efeito, aí vi um video na internet de um cara com compressor MESMO e
eu cheguei comprar um ontem pra mim 120PSI 24L / regulei pra 50PSI de saída
e coloquei o bico apontando para onde o laser iniciava o corte E NADA :(
... Esse seu de 30W, a máquina é parecida com a minha ou é CO2 ou coisa do
tipo ? Acha possível essa minha com apenas o ar, ser possível o corte ( ao
menos na de 3mm ) ?

Em qua., 9 de fev. de 2022 às 16:08, Tribo-2dot0 ***@***.***>
escreveu:

**Comment by Tribo-2dot0** [comment]
Sim, o meu é parecido com esse seu, de diodo, ele tem efeito de 30W por causa da lente colimadora que eles põe que concentra bem o feixe, a saída óptica dele é de 5,5W... esse seu também é saída óptica de 5 ou 5,5W... pelos foruns de laser todos falam ser essencial a assistência a ar... e realmente, se não expulsar a carbonização que vai se formando, fica difícil. Eu quando faço corte é de lâminas de madeira de 1mm, então é sossegado, tenho mais me atido em gravações. Uma madeira fácil de cortar em 3mm é a Balsa.

**Reply by WesleyGarotinho** [reply]
entendi... vlw as dicas aí... vou tentar mais tarde ajustar a distância pra 40mm e ver se é possível finalizar o corte ! obrigado por ora, volto com novidades caso dê certo.
